definitions/private-law/json/CaseEventToFields/EditAndApprove/Page5/CaseEventToFields.json

```diff
@@ -208,41 +208,5 @@
     "PageDisplayOrder": 5,
     "ShowSummaryChangeOption": "Y",
     "FieldShowCondition": "orderType=\"blankOrderOrDirections\" OR orderType=\"other\" OR orderType=\"blank\""
-  },
```

Comment:
This config moved to Page4? any specific reason and other order has been tested?